http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -397,7 +407,7 @@ class RuntimeServiceInterp[F[_]: Parallel](
     as: Ask[F, AppContext]
   ): F[Unit] =
     for {
-      runtimes <- listRuntimes(userInfo, Some(cloudContext), Map.empty)
+      runtimes <- listRuntimes(userInfo, Some(cloudContext), Map.empty, List(RuntimeStatus.Deleted))
```

Comment:
since you have the two different signatures for `listRuntimes`, this could remain unchanged, right? It would default to `List(RuntimeStatus.Deleted)`?
```scala
runtimes <- listRuntimes(userInfo, Some(cloudContext), Map.empty)
```